pkg/kexec-tools/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:db4459afe60aa8affb415cdc370f568fe8bc99bd-dirty
+FROM ${EVE_BUILDER_IMAGE} AS build
```

Comment:
Obviously, you can't use -dirty when this is ready to be merged

Reply:
This is rfc only :) the whole procedure of getting the sha ref of the new eve-alpine container is not clear to me yet, so for now I pushed as is.

Reply:
Got it! Then, could you please add `[WIP]` to the PR title? E.g. https://github.com/lf-edge/eve/pull/2607

Reply:
I think you can just remove `-dirty` from the line. To get tag of package you can run this command: `./build-tools/bin/linuxkit pkg show-tag pkg/alpine`. The `-dirty` suffix in the output indicates that you have uncommitted changes in your source tree (specifically in pkg/alpine). But with uncommitted changes it is not possible to track (from PR point of view) what exactly changed there.

Reply:
But @giggsoff put in a PR that fixed that. It won't make it for this, but still is good that it is fixed.

Reply:
Actually, I stand corrected; that can make it in here, if you rebase on latest eve `master`.